lib/Db/SessionMapper.php

```diff
@@ -111,8 +111,13 @@ public function deleteInactiveWithoutSteps(?int $documentId = null): int {
 		}
 
 		$qb = $this->db->getQueryBuilder();
-		$qb->delete($this->getTableName())
-			->where($qb->expr()->in('id', $qb->createFunction($selectSubQuery->getSQL())));
+		$qb->delete($this->getTableName());
+		if ($documentId !== null) {
```

Comment:
This if block could be merged with the one three lines above, no?

Reply:
In theory yes, but i feel it is easier to read if the subquery and query related calls are separated.

Reply:
Ok, I see your point :+1: